lib/Util/Helpers.php

```diff
@@ -95,4 +98,8 @@ static function recursiveTrim($value) {
     return $value;
   }
 
+  static function isMailPoetUpdateAvailable() {
```

Comment:
I think it is not good idea adding this here. It has a dependency on WordPress which we want to inject and adding it this way will make it hard to refactor.

I would suggest:
- moving `MAILPOET_PLUGIN_PATH ` into Env::plugin_path (note that the property already exists but is not filled in initialization or used anywhere else)
- performing the check in-place in Menu class which already has WPFunctions injected. `$data['is_mailpoet_update_available'] = array_key_exists(Env::$plugin_path,  $this->wp->getPluginUpdates())`